libs/brackets-server/embedded-ext/file-extension/main.js

```diff
@@ -129,7 +130,7 @@ define(function (require, exports, module) {
 
                         // Closure to capture the file information
                         reader.onloadend = loaded(file);
-                        reader.readAsBinaryString(file);
+                        reader.readAsArrayBuffer(file);
                     }
```

Comment:
readAsBinaryString => "result" attribute contains raw binary data from the file as a string,
readAsArrayBuffer => "result" aattribute contains an ArrayBuffer representing the file's data.
(https://developer.mozilla.org/en-US/docs/Web/API/FileReader)

Since readAsArrayBuffer contains ArrayBuffer (not string), File.write operation (Bracket API) may file.
Please check.

Reply:
I already check that File.write operation works correctly with readAsArrayBuffer.

Reproduce way(readAsBinaryString):
  1. Click 'import file' menu
  2. Select image file in your PC
  3. Click OK.
  4. Select imported image file
  5. You can check that image file isn't loaded correctly.

Reply:
@HunseopJeong 
Specifying of encoding option does not helpful for this case ?

Reply:
Yes. I tried to add 'binary' encoding option but it also isn't working correclty.